docs/source/markdown/podman-run.1.md

```diff
@@ -913,6 +917,25 @@ where source dir is mounted on) has to have right propagation properties. For
 shared volumes, source mount point has to be shared. And for slave volumes,
 source mount has to be either shared or slave.
 
+If you want to recursively mount a volume and all of it's submounts into a
+container, then you can use the `rbind` option.  By default the bind option is
+used, and submounts of the source directory will not be mounted into the
+container.
+
+Mounting the volume with the `nosuid` options means that SUID applications on
+the volume will not be able to raise privilege. By default volumes
```

Comment:
nit picking.  Might just be me, but I'm not feeling "raise privilege".  Perhaps "change their privilege".